osmosis-1/osmosis.zone_chains.json

```diff
@@ -1417,7 +1417,9 @@
       "explorer_tx_url": "https://explorer.thestratos.org/stratos/transactions/${txHash}",
       "keplr_features": [
         "ibc-go",
-        "ibc-transfer"
+        "ibc-transfer",
+        "eth-address-gen",
+        "eth-key-sign"
```

Comment:
we should have this in order to pass https://github.com/osmosis-labs/osmosis-frontend/blob/210fed0dbb6bb5bf02e4c4ead9e0effd9bf34ae3/packages/stores/src/account/utils.ts#L102 as we have this functionality in keplr, thats why deposits works in keplr and not worked on osmosis.

Reply:
I have tried to move order here https://github.com/osmosis-labs/osmosis-frontend/pull/3899 and deposit working fine (but there is not need to move as with `eth-key-sign` it will work)

Reply:
@JeremyParish69 should we create another PR for `eth-address-gen` and `eth-key-sign` ?

Reply:
I created separated (just in case): https://github.com/osmosis-labs/assetlists/pull/1964

Reply:
Yes, thanks for that PR. I've already set the asset to be listed (instead of preview) for testing, and I've merged this new one of yours just now so we can proceed with testing.

Reply:
@JeremyParish69 coold, thanks